libfuse/mount_test.go

```diff
@@ -603,6 +612,50 @@ func TestCreateExecutable(t *testing.T) {
 	}
 }
 
+func TestMkdirTLF(t *testing.T) {
+	config := libkbfs.MakeTestConfigOrBust(t, "jdoe", "janedoe")
+	defer libkbfs.CheckConfigAndShutdown(t, config)
+	mnt, _, cancelFn := makeFS(t, config)
+	defer mnt.Close()
+	defer cancelFn()
+	p := path.Join(mnt.Dir, PrivateName, "janedoe,jdoe")
+	checkDir(t, path.Join(mnt.Dir, PrivateName), map[string]fileInfoCheck{
+		"jdoe": nil,
+	})
+	if err := os.MkdirAll(p, 0755); err != nil {
+		t.Fatalf("error creating directory %s: %s", p, err.Error())
+	}
+	checkDir(t, path.Join(mnt.Dir, PrivateName), map[string]fileInfoCheck{
+		"jdoe":         nil,
+		"janedoe,jdoe": nil,
+	})
+}
+
+func TestMkdirAndRmdirTLFCommand(t *testing.T) {
+	config := libkbfs.MakeTestConfigOrBust(t, "jdoe", "janedoe")
+	defer libkbfs.CheckConfigAndShutdown(t, config)
+	mnt, _, cancelFn := makeFS(t, config)
+	defer mnt.Close()
+	defer cancelFn()
+	p := path.Join(mnt.Dir, PrivateName, "janedoe,jdoe")
+	checkDir(t, path.Join(mnt.Dir, PrivateName), map[string]fileInfoCheck{
+		"jdoe": nil,
+	})
+	if b, err := exec.Command("/bin/mkdir", "-p", p).CombinedOutput(); err != nil {
+		t.Fatalf("error executing mkdir -p %s: %s (%s)", p, err.Error(), string(b))
+	}
+	checkDir(t, path.Join(mnt.Dir, PrivateName), map[string]fileInfoCheck{
+		"jdoe":         nil,
+		"janedoe,jdoe": nil,
+	})
+	if b, err := exec.Command("/bin/rmdir", p).CombinedOutput(); err != nil {
```

Comment:
I was under the impression that it was `rmdir` command rather than the syscall that did the lookup after removing the directory. (same for `mkdir`). However, I just tested it on my macOS and it turns out it's the syscalls that did those. So I guess `TestMkdirAndRmdirTLFCommand` is actually not necessary and should be merged into `TestMkdirTLF`.